### PR TITLE
Feature/show ledger key balance

### PR DESCRIPTION
**Pull request by gaupoit**
### Summary (Please recap what you changed or fixed)

- Show balances for ledger keys
- Using loading indicator instead of loading message

### Checklist (If you won't pass this checklist please comment why)

-   [x] I removed all commented or unnecessary code.
-   [ ] Provide the screenshots due to UI changed or bug fixed
-   [x] My code has covered these test cases (please list down your tested cases):

- Key has balance

https://user-images.githubusercontent.com/4486806/157405186-19a0dfe6-5377-4fe7-b4d9-25361d0ce932.MP4

- Key doesn't have balance (0)

https://user-images.githubusercontent.com/4486806/157405263-c78c7a02-f114-4535-a01e-30e333909eae.MP4

### CODE REVIEW

File: src/screens/authentication/create_new_wallet/GetPublicKeyScreen/index.tsx [hunk 1]
```diff
@@ -63,19 +65,50 @@ const GetPublicKeyScreen = ({ transport, setTransport }: Props) => {
       if (ledgerPublicKey) {
```
Comment (ThanhNguyen140797) [inline src/screens/authentication/create_new_wallet/GetPublicKeyScreen/index.tsx:65]: Please move `setIsLoading(true);` in line 60 to line 65 (above `if(ledgerPublicKey)`).
Test case: click Connect Ledger, turn on Ledger but not open Casper App in Ledger. Click the ledger on the app screen.

https://user-images.githubusercontent.com/34257375/157459856-0d69b85a-eedc-4132-9e3a-fa85766be155.MP4

File: src/screens/authentication/create_new_wallet/GetPublicKeyScreen/index.tsx [hunk 2]
```diff
@@ -63,19 +65,50 @@ const GetPublicKeyScreen = ({ transport, setTransport }: Props) => {
...
         setError(null);
         const keys = await getListKeys(casperApp, listKeys.length, 5);
-        setListKeys(keys);
+        dispatch(
+          allActions.user.getAccounts(
+            { publicKeys: keys },
+            async (err: any, data: any) => {
+              if (err) {
+                setError(err);
```
Comment (ThanhNguyen140797) [inline src/screens/authentication/create_new_wallet/GetPublicKeyScreen/index.tsx:73]: The `err` variables is an object. The message will show like this.
IMG_3616

File: src/services/apis.ts [hunk 3]
```diff
@@ -18,6 +18,28 @@ function getAccountInformation<T>(params: any): NetworkPromiseResponse<T> {
...
+/**
+ *  Fetch accounts information by passing public keys.
+ * `NetworkPromiseResponse<T>`
+ * @param {any} params - any
+ * @returns A promise.
+ */
+function getAccounts<T>(params: any): NetworkPromiseResponse<T> {
+  return new Promise((resolve, reject) => {
+    network
+      .authorizedRequest('users/', 'POST', params)
```
Comment (ThanhNguyen140797) [inline src/services/apis.ts:30]: Please update this api for testnet. Api reponse: `url: '/users/ not found'`

Comment (gaupoit) [reply src/services/apis.ts:30]: I updated.

File: src/screens/authentication/create_new_wallet/GetPublicKeyScreen/index.tsx (hunk 1 above)
Comment (gaupoit) [reply src/screens/authentication/create_new_wallet/GetPublicKeyScreen/index.tsx:65]: Nice catch. I fixed it in https://github.com/CasperDash/casperdash-mobile-wallet/pull/52/commits/e98f8b7ca8a9674c7e38ddba630c0bb54f977aba

File: src/screens/authentication/create_new_wallet/GetPublicKeyScreen/index.tsx (hunk 2 above)
Comment (gaupoit) [reply src/screens/authentication/create_new_wallet/GetPublicKeyScreen/index.tsx]: Fixed in https://github.com/CasperDash/casperdash-mobile-wallet/pull/52/commits/aeb97d73318455cc27fdbfe33b06966f58dfdad5

**Review by ThanhNguyen140797 — APPROVED** [review]

**Pull request merged**